src/Runtime/OMExternalConstant.inc

```diff
@@ -58,24 +59,27 @@ void checkEndianness(const char constPackIsLE) {
 ///
 /// This function is thread-safe.
 ///
-void omMMapBinaryFile(
+bool omMMapBinaryFile(
     void **constAddr, char *filename, int64_t size, int64_t isLE) {
   checkEndianness(isLE);
   char *fname = filename;
 #ifdef __MVS__
   // Convert the file name to EBCDIC for the open call.
   char *tPath = strdup(fname);
   if (!tPath) {
-    fprintf(stderr, "Error while strdup");
-    return;
+    fprintf(stderr, "Error while strdup: %s", strerror(errno));
+    return false;
   }
   __a2e_s(tPath);
   fname = tPath;
 #endif
 
   if (constAddr == NULL) {
-    perror("Error: null pointer");
-    return;
+    fprintf(stderr, "Error: null pointer.");
+#ifdef __MVS__
+    free(tPath);
+#endif
```

Comment:
@gongsu832 all of these issues come from the fact that `fname` is not in EBCDIC. So I modify the code generation to convert `fname` to EBCDIC during compilation so that `fname` here would be alread in EBCDIC, then there is no need to handle it here anymore.

Reply:
OK that makes things simpler.